### DISCUSSION TITLE
NDI recv object - how best to get it out?

### DISCUSSION THREAD

**Discussion by yatesdr** [DistroAV/DistroAV#878] [Q&A - Support]
I'm investigating adding NDI PTZ support into another project (obs-ptz).

I don't see a direct way to request the recv object for use in other plug-ins, but am not sure where exactly to look either.

Is this possible already, or will it require a fork?   Any guidance or general knowledge of where to look is appreciated.

**Comment by paulpv** [comment]
PTZ is implemented in the NDI Advanced SDK.
The NDI Advanced SDK is not an option for GNU GPL v2 projects such as obs-ndi.

It is basically the same story as wanting to support NDI|HX.
https://github.com/obs-ndi/obs-ndi/discussions/825

**Reply by Trouffman** [reply]
Actually not true anymore. PTZ control are now available as part of the general SDK : https://docs.ndi.video/docs/sdk/ndi-recv#receiver-ptz-control
